<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta content="width=device-width, initial-scale=1.0" name="viewport">

  <title>RECOMB-CCB: Computational Cancer Biology</title>

  <!-- Favicons -->
  <!-- link href="./assets/img/favicon.png" rel="icon"> -->
  <!-- link href="./assets/img/apple-touch-icon.png" rel="apple-touch-icon"> -->

  <!-- Google Fonts -->
  <link rel="preconnect" href="https://fonts.googleapis.com">
  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
  <link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;1,300;1,400;1,600;1,700&family=Amatic+SC:ital,wght@0,300;0,400;0,500;0,600;0,700;1,300;1,400;1,500;1,600;1,700&family=Inter:ital,wght@0,300;0,400;0,500;0,600;0,700;1,300;1,400;1,500;1,600;1,700&display=swap" rel="stylesheet">

  <!-- Vendor CSS Files -->
  <link href="assets/vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
  <link href="assets/vendor/bootstrap-icons/bootstrap-icons.css" rel="stylesheet">
  <link href="assets/vendor/aos/aos.css" rel="stylesheet">
  <link href="assets/vendor/glightbox/css/glightbox.min.css" rel="stylesheet">
  <link href="assets/vendor/swiper/swiper-bundle.min.css" rel="stylesheet">
  <!-- Template Main CSS File -->
  <link href="assets/css/main.css" rel="stylesheet">
 

</head>
<body>
<!-- ======= Header ======= -->
<header id="header" class="header fixed-top d-flex align-items-center">
    <div class="container d-flex align-items-center justify-content-between">
    
      <div>
      <a href="index.html" class="logo d-flex align-items-center me-auto me-lg-0">
        <h1>16th RECOMB-CCB 2024<span>.</span></h1>
      </a>
      <p>April 27-28, 2024, Boston, MA, USA</p>
      </div>
  <nav id="navbar" class="navbar"><ul>
<li><a href="index.html">Home</a></li>
<li><a href="program.html">Program</a></li>
<li><a href="cfp.html">Call for Papers</a></li>
<li><a href="accepted.html">Accepted Submissions</a></li>
<li><a href="https://recomb.org/recomb2024/register.html">Register</a></li>
<li><a href="https://recomb.org/recomb2024/venue_satellites.html">Venue</a></li>
<!--<li><a href="committees.php.html">Committees</a></li>-->
<!--<li><a href="https://recomb.org/recomb2024/">RECOMB</a></li>-->
</ul></nav>

    <i class="mobile-nav-toggle mobile-nav-show bi bi-list"></i>
    <i class="mobile-nav-toggle mobile-nav-hide d-none bi bi-x"></i>
</div>
</header>
<main id="main">
<section class="hero">
<div class="container">
<div class="section-header"><h1>RECOMB-CCB 2024 Program</h1></div>


<div class="row">
<div class="col-lg-10">

<h3>Saturday, April 27</h3>

<figure class="table" style="width:100%;">
<table>
<colgroup><col style="width:15%;"> <col style="width:45%;"> <col style="width:40%;"></colgroup>
<thead>
<tr>
<td><strong>Time</strong></td>
<td><strong>Title</strong></td>
<td><strong>Author(s)</strong></td>
</tr>
</thead>
<tbody>
<tr>
<td>8:30-9:25 AM</td>
<td>Breakfast and registration</td>
<td></td>
</tr>
<tr>
<td>9:25-9:30 AM</td>
<td>Opening remarks</td>
<td></td>
</tr>
<tr>
<td>9:30-10:30 AM</td>
<td>Keynote: TBD</td>
<td>Gad Getz</td>
</tr>
<tr>
<td>10:30-10:45 AM</td>
<td>Coffee break</td>
<td></td>
</tr>
<tr>
<td>10:45-11:05 AM</td>
<td>Orchard: building large cancer phylogenies using stochastic combinatorial search <a href="https://doi.org/10.48550/arXiv.2311.12917">(paper)</a></td>
<td>Ethan Kulman, Rui Kuang and Quaid Morris</td>
</tr>
<tr>
<td>11:05-11:25 AM</td>
<td>A regression based approach to phylogenetic reconstruction from multi-sample bulk DNA sequencing of tumors (paper)</td>
<td>Henri Schmidt and Benjamin Raphael</td>
</tr>
<tr>
<td>11:25-11:45 AM</td>
<td>Sc-TUSV-ext: Single-cell clonal lineage inference from single nucleotide variants (SNV), copy number alterations (CNA) and structural variants (SV) <a href="https://doi.org/10.1101/2023.12.07.570724">(paper)</a></td>
<td>Nishat Anjum Bristy, Xuecong Fu and Russell Schwartz</td>
</tr>
<tr>
<td>11:45-12:05 PM</td>
<td>Accurate integration of single-cell DNA and RNA for analyzing intratumor heterogeneity using MaCroDNA <a href="https://www.nature.com/articles/s41467-023-44014-3">(highlight)</a></td>
<td>Mohammadamin Edrisi, Xiru Huang, Huw Alexander Ogilvie and Luay Nakhleh</td>
</tr>
<tr>
<td>12:05-1:30 PM</td>
<td>Lunch</td>
<td></td>
</tr>
<tr>
<td>1:30-2:30 PM</td>
<td>Keynote: A new mechanism for oncogene amplification</td>
<td>Peter Park</td>
</tr>
<tr>
<td>2:30-3:00 PM</td>
<td>Lightning talks:
<ol>
<li>Ahmad et al. Haplotype-specific karyotypes reconstruction and copy number aberrations/profiling from long reads sequencing data</li>
<li>Ostroverkhova et al. Elucidating rare driver mutations in human cancer</li>
<li>Ma et al. Spatial landscape of malignant pleural and peritoneal mesothelioma tumor immune microenvironment</li>
<li>Vahid et al. Elucidating transcriptional tumor plasticity in pancreatic adenocarcinoma using pooled single cell RNA CRISPR screens</li>
<li>Geuenich et al. Identifying HLA mediated antigen presentation deficiency at single cell resolution using transfer learning</li>
<li>Stöber et al. Intercellular extrachromosomal DNA copy number heterogeneity drives cancer cell state diversity</li>
</ol></td>
<td></td>
</tr>
<tr>
<td>3:00-4:30 PM</td>
<td>Poster session &amp; coffee break</td>
<td></td>
</tr>
<tr>
<td>4:30-4:50 PM</td>
<td>Assigning mutational signatures to individual samples and individual somatic mutations with SigProfilerAssignment <a href="https://doi.org/10.1093/bioinformatics/btad756">(highlight)</a></td>
<td>Marcos Díaz-Gay, Raviteja Vangara, Mark Barnes, Xi Wang, S.M. Ashiqul Islam, Ian Vermes, Stephen Duke, Nithish Bharadhwaj Narasimman, Ting Yang, Zichen Jiang, Sarah Moody, Sergey Senkin, Paul Brennan, Michael R. Stratton and Ludmil B. Alexandrov</td>
</tr>
<tr>
<td>4:50-5:10 PM</td>
<td>A latent variable model for evaluating mutual exclusivity and co-occurrence between driver mutations in cancer (paper)</td>
<td>Ahmed Shuaibi, Uthsav Chitra and Benjamin Raphael</td>
</tr>
<tr>
<td>5:10-5:30 PM</td>
<td>Refphase: Multi-sample phasing reveals haplotype-specific copy number heterogeneity <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011379">(highlight)</a></td>
<td>Thomas Watkins, Emma Colliver, Matthew Huska, Tom Kaufmann, Emilia Lim, Cody Duncan, Kerstin Haase, Peter Van Loo, Charles Swanton, Nicholas McGranahan and Roland Schwarz</td>
</tr>
<tr>
<td>5:30-5:50 PM</td>
<td>Probing omics data via harmonic persistent homology (paper)</td>
<td>Davide Gurnari, Aldo Guzmán-Sáenz, Filippo Utro, Aritra Bose, Saugata Basu and Laxmi Parida</td>
</tr>
</tbody>
</table>
</figure>

<br>

<h3>Sunday, April 28</h3>

<figure class="table" style="width:100%;">
<table>
<colgroup><col style="width:15%;"> <col style="width:45%;"> <col style="width:40%;"></colgroup>
<thead>
<tr>
<td><strong>Time</strong></td>
<td><strong>Title</strong></td>
<td><strong>Author(s)</strong></td>
</tr>
</thead>
<tbody>
<tr>
<td>8:30-9:30 AM</td>
<td>Breakfast and registration</td>
<td></td>
</tr>
<tr>
<td>9:30-10:30 AM</td>
<td>Keynote: Learning the molecular logic of cancer</td>
<td>Eliezer Van Allen</td>
</tr>
<tr>
<td>10:30-10:45 AM</td>
<td>Coffee break</td>
<td></td>
</tr>
<tr>
<td>10:45-11:05 AM</td>
<td>Tissue-adjusted pathway analysis of cancer (TPAC): A novel approach for quantifying tumor-specific gene set dysregulation relative to normal tissue <a href="https://doi.org/10.1371/journal.pcbi.1011717">(highlight)</a></td>
<td>H. Robert Frost</td>
</tr>
<tr>
<td>11:05-11:25 AM</td>
<td>HIPI: Spatially Resolved Multiplexed Protein ExpressionInferred from H&amp;E WSIs (paper)</td>
<td>Ron Zeira, Leon Anavy, Zohar Yakhini, Ehud Rivlin and Daniel Freedman</td>
</tr>
<tr>
<td>11:25-11:45 AM</td>
<td>Novel mechanism-centric network-based algorithm identifies molecular programs of treatment resistance in cancer <a href="https://www.nature.com/articles/s41467-024-44686-5">(highlight)</a></td>
<td>Sukanya Panja, Mihai Truica, Christina Yu, Vamshi Saggurthi, Michael W. Craige, Katie Whitehead, Mayra Tuiche, Aymen Alsaadi, Riddhi Vyas, Shridar Ganesan, Suril Gohel, Frederick Coffman, James Scott Parrott, Songhua Quan, Shantenu Jha, Isaac Kim, Edward Schaeffer, Vishal Kothari, Sarki Abdulkadir and Antonina Mitrofanova</td>
</tr>
<tr>
<td>11:45-12:05 PM</td>
<td>CADRE: Utilizing CAncer Drug Response nEtwork for Drug Synergy Prediction (paper)</td>
<td>Chunguang Lai, Arda Durmaz, Doug Brubaker and Gurkan Bebek</td>
</tr>
<tr>
<td>12:05-1:30 PM</td>
<td>Lunch provided</td>
<td></td>
</tr>
<tr>
<td>1:30-2:30 PM</td>
<td>Keynote: Mathematical modeling of single cell genomics to understand evolutionary histories of tumors</td>
<td>Franziska Michor</td>
</tr>
<tr>
<td>2:30-2:50 PM</td>
<td>KSFinder 2.0, a model for predicting kinases of human phosphorylation sites, and its potential application in inferring dysregulated kiases in case-specific tumors <a href="https://peerj.com/articles/16164/">(highlight)</a></td>
<td>Manju Anandakrishnan, Karen Ross, Chuming Chen, K.Vijay Shanker and Cathy Wu</td>
</tr>
<tr>
<td>2:50-4:00 PM</td>
<td>Poster session &amp; coffee break</td>
<td></td>
</tr>
<tr>
<td>4:00-4:20 PM</td>
<td>Human papillomavirus integration transforms chromatin to drive oncogenesis <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-02926-9">(highlight)</a></td>
<td>Mehran Karimzadeh, Christopher Arlidge, Ariana Rostami, Mathieu Lupien, Scott V. Bratman and Michael M. Hoffman</td>
</tr>
<tr>
<td>4:20-4:40 PM</td>
<td>On the Performance of Clustering in Large-scale Evolutionary Analyses of Single-Cell DNA Copy Number Data (paper)</td>
<td>Yushu Liu and Luay Nakhleh</td>
</tr>
<tr>
<td>4:40-4:45 PM</td>
<td>Closing remarks</td>
<td></td>
</tr>
</tbody>
</table>
</figure>

</div>
</div>

</div></section></main><footer id="footer" class="footer"><div class="container">
       <div class="credits">
       &copy; RECOMB-CCB conference 2024<br>
       Designed with help of <a href="https://bootstrapmade.com/">BootstrapMade</a>
<br>Last modified: 4/8/2024.
</div></div></footer> 
 <div id="preloader"></div>

  <!-- Vendor JS Files -->
  <script src="assets/vendor/bootstrap/js/bootstrap.bundle.min.js"></script>
  <script src="assets/vendor/aos/aos.js"></script>
  <script src="assets/vendor/glightbox/js/glightbox.min.js"></script>
  <script src="assets/vendor/purecounter/purecounter_vanilla.js"></script>
  <script src="assets/vendor/swiper/swiper-bundle.min.js"></script>
  <script src="assets/vendor/php-email-form/validate.js"></script>

  <!-- Template Main JS File -->
  <script src="assets/js/main.js"></script>

  </body></html>
